deploy/kubernetes/base/kustomization.yaml

```diff
@@ -1,7 +1,7 @@
 commonLabels:
-  app: gcp-compute-persistent-disk-csi-driver
+  k8s-app: gcp-compute-persistent-disk-csi-driver
 namespace:
-  default
+  gce-pd-csi-driver
```

Comment:
add to readme

Reply:
I don't think the specific namespace name should be documented as it could become out of sync, but I'll mention that the deployment exists in a separate namespace